worktrolly-ui/src/app/body/tasks/tasks.component.html

```diff
@@ -15,6 +15,7 @@ <h5 class="active nav-link" style="font-weight:bolder">Tasks</h5>
                     </ul>
                 </nav>
             </div>
+
```

Comment:
no changes done in this file please revert it

Reply:
I actually pulled some code in this branch by mistake after I made a pr and thus some changes which were made by @sandeep-v1404 have also come here.